test/middleware/error-handler.test.ts

```diff
@@ -13,7 +13,7 @@ const makeRequest = async (
 ): Promise<WithDataset<Response>> => {
   const context = createContext({ errorListener });
 
-  return runMiddleware(errorHandler(), context, next);
+  return runMiddleware<ReturnType<typeof createContext>>(errorHandler(), context, next);
```

Comment:
Not sure why this is needed (see also https://github.com/libero/article-store/pull/115/commits/cae9366c72fcf4a7655b739332bb77523edf0ef6#r363793050).

Reply:
Worked it out, the type of `next`.